### ISSUE TITLE
YTD Crashing Immediately with Error in 

### ISSUE THREAD

**Issue by chrisspiegl** [ytmdesktop/ytmdesktop#668]
- [x] I understand that **YTMDesktop have NO affiliation with Google or YouTube**.
- [x] I verified that there is no open issue for the same subject.

**Describe the bug**
- Set the preferences so that the systems should go back to where I left off.
- Also adjusted some keybaord bindings, nothing too special, all in the settings panel.
- Set the Trey icon to not show up (I think, but I can't verify that any more because I can't get into the preferences).
- Now, when I open YTMDesktop it instantly crashes with the following error:

`A JavaScript error occurred in the main process`

```
Uncaught Exception:
ReferenceError: error is not defined
at Object.updateImage (/Applications/YouTube Music Desktop App.app/Contents/Resources/app.asar/src/providers/trayProvider.js:219:45)
at IpcMainImpl.<anonymous> (/Applications/YouTube Music Desktop App.app/Contents/Resources/app.asar/main.js:1739:22)
at IpcMainImpl.emit (events.js:310:20)
at WebContents.<anonymous> (electron/js2c/browser_init.js:157:8362)
at WebContents.emit (events.js:310:20)
```

I can't get back in to change any settings, and I also can't listen to music 🙈.

Hope this is a good description, if there are other details needed, I can provide those.

Looking forward to getting this great app working on my Mac again 🌸.

**Environment (please complete the following information):**
 * YTMDesktop version: ***v1.14.0***
 * OS: Mac
 * OS version: 11.3 Beta (20E5196f)
* Installation way: DMG Downoad & Drag & Drop

**Comment by Alipoodle** [comment]
This issue is fixed in #606 and can be temporarily be fixed by disabling the `settings-shiny-tray` 

I don't know Mac particularly well so I am only taking what I can find on this.
But within the YTM Desktop's Install/Data Folder located within:
`~/Library/Application Support/youtube-music-desktop-app` or `~/Library/Preferences/youtube-music-desktop-app`

There should be an file called `config.json`.  In that set the `settings-shiny-tray` and set it to **false** instead of **true**.

**Comment by chrisspiegl** [comment]
Thank you @Alipoodle. That's awesome to hear that it's already fixed in the dev build.

I updated the setting in the config file, that was very helpful of you. For me the file was in the `~/Library/Application Support/youtube-music-desktop-app` and the crashes stopped happening.

Will be looking out for a new release 👍  before checking the shiny tray again 🙈.

Since the workaround is clear ATM and the fix is already en route, I'll close this.
